cloudigrade/api/views.py

```diff
@@ -91,8 +92,8 @@ class DailyConcurrentUsageViewSet(viewsets.GenericViewSet, mixins.ListModelMixin
 
     def get_queryset(self):  # noqa: C901
         """Get the queryset of dates filtered to the appropriate inputs."""
+        user = User.objects.get(id=self.request.user.id)
```

Comment:
Is `self.request.user` already a `User` instance or is it some other object type? Could you just reference it directly in the lines below like `self.request.user.date_joined.date()`?

Reply:
Oh duh, good catch!